### PR TITLE
add keyup event listener to ESC keyCode + remove pauseAutoSlideShowOn…

### PR DESCRIPTION
**Pull request by yakovhaiilo**
accessibility changes:

when visitors hit the Esc key (keyCode === 27), move the focus to the gallery container.
+ pauseOnFocus : remove unnecessary check from blockAutoSlideShow (refactor)

### CODE REVIEW

**Review by nirnaor — APPROVED** [review]

File: packages/gallery/src/components/gallery/proGallery/slideshowView.js [hunk 1]
```diff
@@ -1326,7 +1326,6 @@ class SlideshowView extends GalleryComponent {
       should = true;
     } else if (
       isInFocus &&
-      styleParams.pauseAutoSlideshowOnHover &&
```
Comment (yhattav) [inline packages/gallery/src/components/gallery/proGallery/slideshowView.js:1329]: why is this deleted?
